spec/models/report_spec.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Report, type: :model do
+  describe '#editable' do
+    let(:user) do
+      User.create!(
+        name: 'alice',
+        email: 'alice@example.com',
+        password: '123456',
+        password_confirmation: '123456'
+      )
+    end
+    let(:another_user) do
+      User.create!(
+        name: 'bob',
+        email: 'bob@example.com',
+        password: '123456',
+        password_confirmation: '123456'
+      )
+    end
+    let(:report) do
+      user.reports.create!(
+        content: 'aliceの日報です、日報を書きます。今回は編集が可能かのテストのために書きました。',
+        title: '今日のレポート'
+      )
+    end
+    context 'ユーザーがレポートを所有している' do
+      it '編集できる' do
+        expect(report.editable?(user)).to be_truthy
+      end
+    end
+    context 'ユーザーがレポートを所有していない' do
+      it '編集できない' do
+        expect(report.editable?(another_user)).to be_falsey
+      end
+    end
+  end
+  describe '#created_on' do
+    let(:user) do
+      User.create!(
+        name: 'alice',
+        email: 'alice@example.com',
+        password: '123456',
+        password_confirmation: '123456'
+      )
+    end
+    let(:report) do
+      user.reports.create!(
+        content: 'aliceの日報です、日報を書きます。今回はcreated_onがしっかり日付を取得できているのか確かめるテストのために書きました。',
+        title: '今日のレポート'
+      )
+    end
+    it '作られた日付を取得' do
+      report.created_at = 'Tue, 24 Jun 2024 14:33:07.146486000 JST +09:00'
```

Comment:
```suggestion
      report.created_at = "2024-06-24 14:33".in_time_zone
```

みたいにした方が見やすいですし、文字列から日時型への暗黙的な自動変換も発生しないのでベターです